### PR TITLE
Nice enhancements

### PR DESCRIPTION
**Pull request by sursly**
to almost every letter, number, and some other punctuation/diacritics

### CODE REVIEW

**Comment by sursly** [comment]
Check it out @alerque ☝️

**Review by alerque — CHANGES_REQUESTED** [review]
Everything looks good to me. Just the minor issue of a dead URL in the readme. Also note perhaps this URL should be added somewhere: https://www.theleagueofmoveabletype.com/league-mono

If all looks good after this merges it might be fair to tag another release.

File: README.md
Comment (sursly) [reply README.md:10]: Ah, it should be: http://demos.tyfromtheinternet.com/leaguemonovariable/

Comment (alerque) [reply README.md:10]: That is the demo link, and is also linked later in the README. This is just the callout for you as a person, so perhaps linking your domain would be more to the point. Your call though, it's your publicity I'm vouching for here ;-)

Comment (sursly) [reply README.md:10]: Oh, right! OK, whatever you think is best, I was confused. Just wanted to help out. LMK if you need anything.

Comment (alerque) [reply README.md:10]: Roger that. Since I don't have permission to change things in this PR (since it is from your fork's default branch instead of a feature branch) I'm going to go ahead and merge as-is and then fix up the URL issue.

**Pull request merged**
